### ISSUE TITLE
[Video Module] - Support bucket sizes and true sizes for Video

### ISSUE THREAD

**Issue by karimMourra** [prebid/Prebid.js#9863]
## Type of issue
Feature Request

## Description
The Video Module reports the real size of the player. Since most players are dynamic, the size may vary based on the page and container width. Unfortunately, most SSPs target very specific player sizes, meaning that a 495x655 size will not get picked up by an SSP that targets 480x640.

To help publishers, we want to offer a config value `use true size` which will send the real size when set to true, or the nearest standard targeted size when set to false. We will also send both the real and the bucket size in the `video.ext` field for transparency.

**Comment by karimMourra** [comment]
@spormeon I apologize, I do not understand the question. Could you please rephrase ?

**Comment by spormeon** [comment]
you changed the module to allow mediaTypes: { video to be set, so you can set the size there already?


![NewVideoModuleNeuwoGolf1_html_—_Prebid_Publishers__Workspace_](https://user-images.githubusercontent.com/8503167/234983764-c858ee26-34da-4967-a7b5-e495d7a27f4b.png)

**Comment by karimMourra** [comment]
Yes! a size specified by the publisher will take precedence. But the idea of the feature is to help publishers who do not know the size of the player at auction time. Realistically, most publishers say 640x480, and the actual size is different. This is because player size is often variable, as it will adjust to the width of the page, or the container in which it is embedded, etc. So for publishers wishing to know the real size, and the closest standardly targeted size, this feature could be useful. 

For example,  if the player is 301x 234 at bid request time, we could report the real size, or 320x240 which would be the closest standard targeted size.

**Comment by spormeon** [comment]
ok got ye, so your gonna work in logic to send 320x240 to only those bidders that support that size and send nothing to the e.g 5/10 that dont? or is there gonna be an exaustable list of what every bidder will/ wont bid on?

**Comment by bweiss-freestar** [comment]
Karim, to summarize, is it best to send the closest standard size in the request to ssp's rather than the true size? e.g. 640x480v rather than whatever the true size is at a 16:9 player ratio based on the responsive resizing of the player?

**Comment by dgirardi** [comment]
Proposal:

 - setting config `videoModule: {sizeBuckets: [[200, 300], [400, 600], ...]}` would send the nearest bucket size
 - when set, `imp.video.ext.true_size` would reflect the actual size
 
What do you think @karimMourra ?

**Comment by karimMourra** [comment]
sounds good. is `imp.video.ext.true_size` common ? I've never seen it before.
